### ISSUE TITLE
Question : How to add integration with Stripe Elements (v3)

### ISSUE THREAD

**Issue by callsecfrance** [muhammed/vue-interactive-paycard#46]
Hello, is it possible to integrate Stripe with this page (while keeping the animations) ?
I tried on my side and I did not succeed...

If anyone was successful, feel free to share.
Thank you 👍

**Comment by bluestarstudios** [comment]
Agreed! This is such a GORGEOUS thing. Adding a Stripe Elements integration option would be fantastic!
